src/main/java/teammates/common/util/Config.java

```diff
@@ -13,6 +13,10 @@
  */
 public final class Config {
 
+    // Normally, the following property values are loaded from build.properties file.
+    // When in dev environment and they appear in build-dev.properties,
+    // they are overridden by the values in build-dev.properties.
```

Comment:
This comment is not in the right place. The class doesn't really care how the fields are set. This should be placed in the `static { ... }` block where all the fields are set.